path_rotate.go

```diff
@@ -70,7 +70,12 @@ func (b *backend) pathRotateCredentials() []*framework.Path {
 }
 
 func (b *backend) pathRotateRootCredentialsUpdate(ctx context.Context, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
-	return nil, b.rotateRootCredential(ctx, req)
+	err := b.rotateRootCredential(ctx, req)
+	if err != nil {
+		b.Logger().Error("failed to rotate root credential:", "rotationID", req.Path, "err", err.Error())
```

Comment:
This is leaking rotation manager implementation details. What happens if rotation manager changes the format or details of the rotation ID? Should we use `req.RotationID` instead of `req.Path`?

Reply:
Incidentally, I'm not sure rotationID is populated if the request comes directly from the RotationManager, in which case
1) That's only true inside the function registered as the RotateCredential function, and
2) We're logging general success/failure there already, so we want to avoid logging here